Jenkinsfile

```diff
@@ -291,33 +313,32 @@ pipeline {
         // Misc tests
         stage("Aux tests"){
             parallel{
-                stage('Hip clang debug COMGR') {
-                    // WORKAROUND for COMGR Vega10 testing problem. Should be "vega".
-                    agent{ label rocmnode("vega20") }
+                stage('Hip Debug COMGR') {
+                    agent{ label rocmnode("vega") }
```

Comment:
so the comgr test is no longer restricted to vega20?

Reply:
Yes. That was a bug that prevented COMgr to work with Vega10 correctly. Fixed by #529